docs/dev/sstable-scylla-format.md

```diff
@@ -129,6 +129,7 @@ guaranteed to be disjoint (non-overlapping) in their partition keys.
         row_size = be32(2)          // row size, in bytes
         cell_size = be32(3)         // cell size, in bytes
         rows_in_partition = be32(4) // number of rows in a partition
+        items_in_collection = be32(5) // number of items in a collection
```

Comment:
Need to update `large_data_type`

Reply:
it is updated. In the last hunk of this patch.

Reply:
It also appears a few rows above.

Reply:
ok, got it now

Reply:
fixed in v5